Reproduce-MuSIC-paper-results.ipynb

```diff
@@ -24,9 +24,9 @@
    "outputs": [],
    "source": [
     "%%bash\n",
-    "conda create -n music python=3.6.2 anaconda\n",
```

Comment:
I removed anaconda here, @cwinsnes and I tested and it seems work, but please confirm it.

Reply:
Note that the `anaconda` part is already gone in the `.md` version of the step-by-step. If we keep `anaconda` here we need to readd it in the other installation instructions.

Keeping the installation instructions in one place sounds like a good idea.